### ISSUE TITLE
Angular is not defined?

### ISSUE THREAD

**Issue by smlombardi** [ricardofbarros/linter-js-standard#99]
Every definition of angular in my project is flagged as not defined, as in

`````` js
(function () {
  'use strict'

  // app constants
  angular.module('app').constant('config', {
    debug: true
  })
})()```

Following instructions elsewhere, I installed `eslint-plugin-angular`, I created an .eslintrc file with 
``````

{
  "extends": "angular"
}

```
and I even added to package.json:
```

  "standard": {
    "globals": ["_", "angular", "s"]
  }

```
This works for sublime's standard linter, but not this one.  Am I doing it incorrectly?
```

**Comment by jagged3dge** [comment]
You could, in essence, do:
```
/* global angular */

(function () {
  'use strict'

  // app constants
  angular.module('app').constant('config', {
    debug: true
  })
})()
```
However, this solution is on a per-file basis.

**Comment by ricardofbarros** [comment]
Hello @smlombardi ,

what you need to do is, like you did is add to your project package.json: 

```json
...
  "standard": {
    "globals": [
      "angular"
    ]
  },
...
```

__and tick the checkbox of `Honor style settings` on the package settings.__

Could you do that and after that give me some feedback?
